docs/src/estimators_cheatsheet.md

```diff
@@ -0,0 +1,298 @@
+```
+
+### 1. The Asymptotically Linear Term
+
+```math
+\mathbb{P}_n\phi_{\mathbb{P}}(Z)
+```
+
+By the central limit theorem, it is asymptotically normal with variance ``Var[\phi]/n``, it is used to build confidence interval.
+
+### 2. The Bias Term
+
+```math
+- \mathbb{P}_n\phi_{\mathbb{\hat{P}}}(Z)
+```
+
+This is the term both the One-Step estimator and the Targeted Maximum-Likelihood estimator deal with.
+
+### 3. The Empirical Process Term
+
+```math
+(\mathbb{P}_n − \mathbb{P})(ϕ_{\mathbb{\hat{P}}}(Z) − \phi_{\mathbb{P}}(Z))
+```
+
+This can be shown to be of order ``o_{\mathbb{P}}(\frac{1}{\sqrt{n}})`` if ``\phi_{\hat{\mathbb{P}}}`` converges to ``\phi_{\mathbb{P}}`` in ``L_2(\mathbb{P})`` norm, that is:
+
+```math
+\int (\phi_{\hat{\mathbb{P}}}(Z) - \phi_{\mathbb{P}}(Z) )^2 d\mathbb{P}(Z) = o_{\mathbb{P}}(\frac{1}{\sqrt{n}})
+```
+
+and, any of the following holds:
+
+- ``\phi`` (or equivalently its components) is [[Donsker](https://en.wikipedia.org/wiki/Donsker_classes)], i.e., not too complex.
+- The estimator is constructed using sample-splitting (see cross-validated estimators).
+
+### 4. The Second-Order Remainder Term
+
+```math
+R_2(\mathbb{\hat{P}}, \mathbb{P})
+```
+
+This term is usually more complex to analyse. However, for the counterfactual mean, it can be shown that if ``g(W,T) \geq \frac{1}{\eta}`` (positivity constraint):
+
+```math
+|R_2(\mathbb{\hat{P}}, \mathbb{P})| \leq \eta ||\hat{Q}_{n, Y} - Q_{Y}|| \cdot ||\hat{g}_{n} - g||
+```
+
+and thus, if the estimators ``\hat{Q}_{n, Y}`` and ``\hat{g}_{n}`` converge at a rate ``o_{\mathbb{P}}(n^{-\frac{1}{4}})``, the second-order remainder will be ``o_{\mathbb{P}}(\frac{1}{\sqrt{n}})``. This is the case for many popular estimators like random forests, neural networks, etc...
+
+## Asymptotic Linearity and Inference
+
+According to the previous section, the OSE and TMLE will be asymptotically linear with efficient influence curve the gradient ``\phi``:
+
+```math
+\sqrt{n}(\hat{\Psi} - \Psi) = \frac{1}{\sqrt{n}} \sum_{i=1}^n \phi(Z_i) + o_{\mathbb{P}}(\frac{1}{\sqrt{n}})
+```
+
+By the [Central Limit Theorem](https://en.wikipedia.org/wiki/Central_limit_theorem) and [Slutsky's Theorem](https://en.wikipedia.org/wiki/Slutsky%27s_theorem) we have:
+
+```math
+\sqrt{n}(\hat{\Psi} - \Psi) \leadsto \mathcal{N}(0, Std(\phi))
+```
+
+Now, if we consider ``S_n = \hat{Std}(\phi_{\hat{\mathbb{P}}})`` as a consistent estimator for ``Std(\phi)``, we have by Slutsky's Theorem again, the following pivot:
+
+```math
+\frac{\sqrt{n}(\hat{\Psi} - \Psi)}{S_n} \leadsto \mathcal{N}(0, 1)
+```
+
+which can be used to build confidence intervals:
+
+```math
+\underset{n \to \infty}{\lim} P(\hat{\Psi}_n - \frac{S_n}{\sqrt{n}}z_{\alpha} \leq \Psi \leq \hat{\Psi}_n + \frac{S_n}{\sqrt{n}}z_{\alpha}) = 1 - 2\alpha
+```
+
```

Comment:
Suggestion: "Here, ``z_{\alpha}`` denotes the ``\alpha``-quantile function of the standard normal distribution.